guides/source/active_record_basics.md

```diff
@@ -220,134 +265,196 @@ The application will have to use the [`id_value`][] alias attribute to access th
 
 NOTE: If you try to create a column named `id` which is not the primary key,
 Rails will throw an error during migrations such as:
-`you can't redefine the primary key column 'id' on 'my_products'.`
+`you can't redefine the primary key column 'id' on 'my_books'.`
 `To define a custom primary key, pass { id: false } to create_table.`
 
 CRUD: Reading and Writing Data
 ------------------------------
 
 CRUD is an acronym for the four verbs we use to operate on data: **C**reate,
 **R**ead, **U**pdate and **D**elete. Active Record automatically creates methods
-to allow an application to read and manipulate data stored within its tables.
+to allow you to read and manipulate data stored in your application's database tables.
+
+Active Record makes it seamless to perform CRUD operations by using these high-level methods that abstract away database access details. Note that all of these convenient methods result in SQL statement(s) that are executed against the underlying database.
+
+The examples below show a few of the CRUD methods as well as the resulting SQL statements.
 
 ### Create
 
 Active Record objects can be created from a hash, a block, or have their
 attributes manually set after creation. The `new` method will return a new
 object while `create` will return the object and save it to the database.
 
-For example, given a model `User` with attributes of `name` and `occupation`,
+For example, given a `Book` model with attributes of `title` and `author`,
 the `create` method call will create and save a new record into the database:
 
 ```ruby
-user = User.create(name: "David", occupation: "Code Artist")
+book = Book.create(title: "The Lord of the Rings", author: "J.R.R. Tolkien")
```

Comment:
Perhaps we should add another line to the example that inspects the object to show that the `id` is set to an integer, to contrast it with the same example in the `#new` example below:

```suggestion
book = Book.create(title: "The Lord of the Rings", author: "J.R.R. Tolkien")

book.inspect
# "#<Book id:1 title: \"The Lord of the Rings\" author: \"J.R.R. Tolkien\" created_at: nil, updated_at: nil>"
```

Reply:
Updated!